agentstack/main.py

```
@@ -199,17 +202,33 @@ def main():
             parser.print_help()
     except Exception as e:
         update_telemetry(telemetry_id, result=1, message=str(e))
-        print(term_color("An error occurred while running your AgentStack command:", "red"))
-        print(e)
-        sys.exit(1)
+        raise e
```

We can re-raise the exception here and still handle the print in `main` below